### PR TITLE
UP-4643 updates commons collections

### PR DESCRIPTION
**Pull request by vertein**
Updates commons collections library, while updating portlets that were patched to update the commons collections library.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -150,7 +150,7 @@
         <cobertura-maven-plugin.version>2.6</cobertura-maven-plugin.version>
         <commons-cli.version>1.2</commons-cli.version>
         <commons-codec.version>1.9</commons-codec.version>
-        <commons-collections.version>3.2.1</commons-collections.version>
+        <commons-collections.version>3.2.2</commons-collections.version>
```
Comment (apetro) [inline pom.xml]: a5be8b0f7de1b8dbcfbf7bbdc31f017c124d185b on `master` already did this bit?

Comment (vertein) [inline pom.xml]: Rebased branch to solve this.

**Comment by blamonet** [comment]
:+1:  Looks good. I implemented this change locally.

**Comment by drewwills** [comment]
:+1:

**Pull request merged**